scripts/absolute_coordinates.pl

```diff
@@ -37,7 +37,10 @@
 close SIZE;
 my %finalSizes = ();
 my $sizeSoFar = 0;
-use chrSort;
+use FindBin;
+use File::Spec;
+use lib File::Spec->catdir($FindBin::RealBin, '/../');
+use SortChr;
```

Comment:
By the way, I think there is a misspell in the module name in this script. the module name is `SortChr`, not `chrSort`.